data/tlg0062/tlg019/tlg0062.tlg019.perseus-eng5.xml

```diff
@@ -0,0 +1,1378 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">The Cock</title>
+ <author>Lucian of Samosata</author>
+<editor role="translator">Emily James Smith</editor>
+ <funder>National Endowment for the Humanities</funder>
+ <principal>Gregory Crane</principal>
+ <respStmt>
+  <persName>Gregory Crane</persName>
+<resp>Digital conversion and editing</resp>
+ </respStmt>
+ </titleStmt>
+<publicationStmt>
+<authority>Perseus Digital Library</authority>
+<idno type="filename">tlg0062.tlg019.perseus-eng5.xml</idno>
+<availability>
+ <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+</availability>
+<date type="release">2023-08-01</date>
+<publisher>Harvard College Library</publisher>
+<pubPlace>United States</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl>
+ <biblStruct>
+ <monogr>
+<title xml:lang="eng">Selections from Lucian</title>
+  <editor role="translator">Emily James Smith</editor>
+  <author ref="urn:cts:greekLit:tlg0062">Lucian of Samosata</author>
+  <imprint>
+   <publisher>Harper and Brothers</publisher>
+	<pubPlace>New York</pubPlace>
+	<date>1892</date>
+  </imprint>
+ </monogr>
+  <ref target="https://hdl.handle.net/2027/hvd.hw3j0n">The Hathi Trust</ref>
+ </biblStruct>
+</listBibl>
+</sourceDesc>
+</fileDesc>
+<encodingDesc>
+ <p>Text encoded in accordance with the latest EpiDoc standards</p>
+ <refsDecl n="CTS">
+  <cRefPattern matchPattern="(\w+)"
+   replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">
+   <p>This pointer pattern extracts section.</p>
+  </cRefPattern>
+ </refsDecl>
+</encodingDesc>
+<profileDesc>
+<langUsage>
+<language ident="grc">Greek</language>
+<language ident="lat">Latin</language>
+</langUsage>
+</profileDesc>
+<revisionDesc>
+<change/>
```

Comment:
I was just noting now that in all those empty change logs you should probably put a first value of something like who="Greg Crane" when="Now" and a description like "Created initial file in repository while spending too much time on beach doing translation work and driving Alison crazy" or something like that!

Reply:
A very useful starting point ...